go.mod

```diff
@@ -367,6 +367,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require gopkg.in/mcuadros/go-syslog.v2 v2.3.0
```

Comment:
I'm a bit worried that this might impact downstream (i.e. operator) build. But I wasn't able to validate it.

Can you try the following:
1. create a go.mod file in `qa-tests-backend/test-images/syslog`
2. Remove the changes here and an in go.sum
3. Use this in that new go.mod file:
```
module github.com/stackrox/stackrox/qa-tests-backend/test-images/syslog

go 1.18

require gopkg.in/mcuadros/go-syslog.v2 v2.3.0
```
4. cd to `qa-tests-backend/test-images/syslog` and run `go mod tidy`